examples/llama/test.zig

```diff
@@ -102,25 +100,27 @@ fn testImplementation(
     try zml.testing.testLayer(platform, buffer_store, "layers.0.input_layernorm", llama.model.layers[0].input_layernorm, llama_weights.model.layers[0].input_layernorm, 1e-2);
     try zml.testing.testLayer(platform, buffer_store, "layers.0.post_attention_layernorm", llama.model.layers[0].post_attention_layernorm, llama_weights.model.layers[0].post_attention_layernorm, 1e-2);
 
-    {
-        const test_case = "layers.0.self_attn";
-        std.log.info("Testing {s}", .{test_case});
-        // Small wrapper to explicitly tag the input, and ignore the extra arguments used in HF implementation.
-        const SelfAttnPrefill = struct {
-            inner: llama_mod.SelfAttn,
-
-            pub fn forward(self: @This(), x_: Tensor) struct { Tensor, llama_mod.KvCache } {
-                return self.inner.forward(x_.withTags(.{ .b, .s, .d }), null, null);
-            }
-        };
-
-        try zml.testing.testLayer(
-            platform,
-            buffer_store,
-            "layers.0.self_attn",
-            SelfAttnPrefill{ .inner = llama.model.layers[0].self_attn },
-            .{ .inner = llama_weights.model.layers[0].self_attn },
-            1e-3,
-        );
-    }
+    // needs rework now that we don't support null tensors anymore
```

Comment:
what's up with  those tests ?

Reply:
idk. but the bottom portion does not work anymore. if anyone cares, he can fix it. if not, I'll remove it. the test needs activations that are not provided (not downloaded from anywhere) anyway so they only make sense to the author as of now.

Reply:
i mean now prefill and forward are the same, a dedicated prefill test isn't really needed imho. so i removed the commented out test